sound/soc/sof/loader.c

```diff
@@ -310,6 +310,8 @@ static void sof_get_windows(struct snd_sof_dev *sdev)
 	u32 outbox_size = 0;
 	u32 stream_size = 0;
 	u32 inbox_size = 0;
+	u32 debug_size = 0;
```

Comment:
is there a typo in the commit message? "in case of dsp ops" or "in case of dsp oops"?